src/compiler/compile/nodes/Text.ts

```diff
@@ -42,4 +43,35 @@ export default class Text extends Node {
 
 		return parent_element.namespace || elements_without_text.has(parent_element.name);
 	}
+
+	validate() {
+		const { data, component, parent } = this;
+
+		// https://github.com/mathiasbynens/emoji-regex/blob/master/src/index.js
+		const emoji_regex =/<% emojiSequence %>|\p{Emoji_Presentation}|\p{Emoji}\uFE0F|\p{Emoji_Modifier_Base}/gu;
```

Comment:
Even though it's now EOL, Node 8 is still officially supported by Svelte 3, which means we can't use regex features like this - and this is failing CI which is running on Node 8, 10, 12, and 14.

There's also a previous PR #4411 for this that's been languishing for a little bit, but which is compatible with Node 8.

Reply:
Oops - completely forgot to check for existing PRs. In that case I'll close this one. Thanks :)